docs/userGuide/syntax/links.mbdf

```diff
@@ -44,7 +44,7 @@ You may link to markdown files using its original extension (**.md**) as it will
 If you wish to disable the auto-conversion, you may use the `no-convert` attribute in your link. 
 
 {{ icon_example }}
-`Click [here](index.md) {.no-convert}`
+`Click [here](index.md){no-convert}`
```

Comment:
Just a suggestion, should we leave a space before the '{' to ensure that it is consistent with the user guide on [Classes, Attributes & Identifiers](https://markbind-master.netlify.app/userguide/formattingcontents#classes-attributes-and-amp-identifiers)?

Reply:
Hi @jonahtanjz! The library that we are using, `markdown-it-attrs` actually parses inline curly braces a little differently. There must not be a space in between in this example, otherwise it wouldn't work. The space I believe is only for block-level elements. 

But to be honest, I think our user guide probably isn't that clear on this aspect. For example, the phrase `Some other types of markdown have different placements of the curly group {...}.` from the user guide doesn't quite stand out but it's an important piece of information. 

I was thinking maybe we can relook at this portion of the user guide sometime in the future to improve the clarity. But what do you think about this section of the user guide? :-)

Reply:
> Hi @jonahtanjz! The library that we are using, markdown-it-attrs actually parses inline curly braces a little differently. There must not be a space in between in this example, otherwise it wouldn't work. The space I believe is only for block-level elements.

Ahh alright thanks for clarifying 👍

> But what do you think about this section of the user guide? :-)

Yup agree with you that this portion is slightly vague. Perhaps can draw more attention towards this line and also include a few examples of the different placements of `{...}` so that users can have a clearer idea of this.